Jenkinsfile

```diff
@@ -96,11 +92,10 @@ pipeline {
                 timeout(time: 3, unit: 'HOURS')
             }
             steps {
-                // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn install -B -pl $ITESTS -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn install -B -DskipITs=false -pl $ITESTS -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
```

Comment:
does this explicitly need to be set to false?  or can it just be removed?

Reply:
i can test this, it may work if we dont set a value for this